backend/addcorpus/models.py

```diff
@@ -57,6 +57,10 @@ class Corpus(models.Model):
         help_text='whether the configuration of this corpus is determined by a Python '
             'module (some features are only available for Python-based corpora)',
     )
+    date_created = models.DateField(
+        auto_now_add=True,
+        help_text='date on which the corpus was added to the database',
+    )
 
```

Comment:
You could also do:
```suggestion
class Meta:
        ordering = ["date_created"]
```
instead of explicitly calling `order_by` in the `views.py`. Or do you want a different ordering in different situations?